src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -99,10 +99,14 @@ public function __construct(ClassMetadataFactoryInterface $classMetadataFactory
         $this->nameConverter = $nameConverter;
         $this->defaultContext = array_merge($this->defaultContext, $defaultContext);
 
-        if (\is_array($this->defaultContext[self::CALLBACKS] ?? null)) {
+        if (\array_key_exists(self::CALLBACKS, $this->defaultContext)) {
```

Comment:
why not isset?

Reply:
same everywhere :)

Reply:
thanks joel for the adjusting.

i started favoring array_key_exists, because isset would not give you an error if e.g. you did a refactoring and $this->defaultContext does not exist anymore. its a fragile thing, whereas array_key_exist is much mor explicit in what you actually want to check.